lib/src/server_messages.dart

```diff
@@ -133,6 +133,19 @@ class DataRowMessage extends ServerMessage {
   String toString() => "Data Row Message: ${values}";
 }
 
+class NotificationResponseMessage extends ServerMessage {
+  int processId;
+  String channel;
+  String payload;
+
+  void readBytes(Uint8List bytes) {
+    var view = new ByteData.view(bytes.buffer, bytes.offsetInBytes);
+    processId = view.getUint32(0);
+    channel = UTF8.decode(bytes.sublist(4, bytes.indexOf(0, 4)));
```

Comment:
Bah, this is my fault - I think your earlier implementation was just fine. If there really is a need to optimize this, there are a few other places to do it too, and that can wait. Can you revert back to what you had here? And then making the processID public on the connection, I think we're all set.